libs/auth/src/angular/change-password/change-password.component.ts

```diff
@@ -0,0 +1,203 @@
+import { firstValueFrom, map } from "rxjs";
+
+import { ChangePasswordService } from "@bitwarden/auth/common";
+import { PolicyService } from "@bitwarden/common/admin-console/abstractions/policy/policy.service.abstraction";
+import { MasterPasswordPolicyOptions } from "@bitwarden/common/admin-console/models/domain/master-password-policy-options";
+import { AccountService } from "@bitwarden/common/auth/abstractions/account.service";
+import { MasterPasswordApiService } from "@bitwarden/common/auth/abstractions/master-password-api.service.abstraction";
+import { PasswordRequest } from "@bitwarden/common/auth/models/request/password.request";
+import { getUserId } from "@bitwarden/common/auth/services/account.service";
+import { FeatureFlag } from "@bitwarden/common/enums/feature-flag.enum";
+import { InternalMasterPasswordServiceAbstraction } from "@bitwarden/common/key-management/master-password/abstractions/master-password.service.abstraction";
+import { ConfigService } from "@bitwarden/common/platform/abstractions/config/config.service";
+import { I18nService } from "@bitwarden/common/platform/abstractions/i18n.service";
+import { MessagingService } from "@bitwarden/common/platform/abstractions/messaging.service";
+import { EncString } from "@bitwarden/common/platform/models/domain/enc-string";
+import { SyncService } from "@bitwarden/common/platform/sync";
+import { UserId } from "@bitwarden/common/types/guid";
+import { UserKey } from "@bitwarden/common/types/key";
+import { ToastService } from "@bitwarden/components";
+import { KeyService } from "@bitwarden/key-management";
+
+import {
+  InputPasswordComponent,
+  InputPasswordFlow,
+} from "../input-password/input-password.component";
+import { PasswordInputResult } from "../input-password/password-input-result";
+
+@Component({
+  standalone: true,
+  selector: "auth-change-password",
+  templateUrl: "change-password.component.html",
+  imports: [InputPasswordComponent],
+})
+export class ChangePasswordComponent implements OnInit {
+  @Input() inputPasswordFlow: InputPasswordFlow = InputPasswordFlow.ChangePassword;
+
+  email?: string;
+  masterPasswordPolicyOptions?: MasterPasswordPolicyOptions;
+  userkeyRotationV2 = false;
+  formPromise?: Promise<any>;
+
+  constructor(
+    private accountService: AccountService,
+    private changePasswordService: ChangePasswordService,
+    private configService: ConfigService,
+    private i18nService: I18nService,
+    private keyService: KeyService,
+    private masterPasswordApiService: MasterPasswordApiService,
+    private masterPasswordService: InternalMasterPasswordServiceAbstraction,
+    private messagingService: MessagingService,
+    private policyService: PolicyService,
+    private toastService: ToastService,
+    private syncService: SyncService,
+  ) {}
+
+  async ngOnInit() {
+    this.userkeyRotationV2 = await this.configService.getFeatureFlag(FeatureFlag.UserKeyRotationV2);
+
+    this.email = await firstValueFrom(
```

Comment:
Issue: This uses firstValueFrom twice, once to get the active account, and to get the email from that, and once to get the active account, and to get the userId from that. 
It is easier to reason about these being in sync and consistent if we get the active account once, and get the userid and email from this one value.

```suggestion
    const activeAccount =  await firstValueFrom(this.accountService.activeAccount$);
    const userId = activeAccount.id;
    this.email = activeAccount.email;
```